pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -208,6 +220,11 @@ stdenv.mkDerivation rec {
   # --force-rpath prevents changing RPATH (set above) to RUNPATH.
   postFixup = ''
     addOpenGLRunpath --force-rpath {$out,$lib}/lib/lib*.so
+    ${lib.optionalString (lib.versionAtLeast version "11") ''
+      addOpenGLRunpath $out/cuda_sanitizer_api/compute-sanitizer/*
+      addOpenGLRunpath $out/cuda_sanitizer_api/compute-sanitizer/x86/*
+      addOpenGLRunpath $out/target-linux-x64/*
+    ''}
```

Comment:
same nit re using `+ lib.optionalString`

Reply:
Fixed in cbc2d73